composer.json

```diff
@@ -45,7 +45,7 @@
     ],
     "homepage": "https://www.doctrine-project.org/",
     "require": {
-        "php": "^7.4 || ~8.0.0 || ~8.1.0",
+        "php": "^7.4 || ~8.0.0 || ~8.1.0 || ~8.2.0",
         "ext-mongodb": "*",
         "doctrine/doctrine-laminas-hydrator": "^3.1.0",
         "doctrine/doctrine-module": "^5.2.1",
```

Comment:
Could you please bump this to 5.3.0? This reduces complexity for maintainers, as less combinations of the two modules are allowed then.

Reply:
Done!